weatherrouting/routers/linearbestisorouter.py

```diff
@@ -26,26 +26,43 @@ class LinearBestIsoRouter (Router):
 	}
 
 	def route (self, lastlog, time, start, end):
```

Comment:
I don't understand what you are doing here; can you please explain?

Reply:
Ok; can you provide also a test which match this case?

Reply:
https://github.com/dakk/libweatherrouting/pull/3/commits/e758b5fd3efe5ce67647103c9d690e49255ac7f5